cfgov/scripts/enforcement_transfer.py

```diff
@@ -0,0 +1,191 @@
+    url = page.get_url()
+
+    if not url or 'policy-compliance/enforcement/actions' not in url:
+        return
+
+    keys = vars(page)
+
+    old_categories = page.categories.all()
+
+    if not len(old_categories):
+        return errors.append(f'No assigned categories for {url}')
+
+    category = CFGOVPageCategory(name=old_categories[0].name)
+
+    tags = page.tags.all()
+    sidefoot = get_stream_data(page, 'sidefoot')
+    sidefoot_fields = []
+
+    court = ''
+    docket_number = None
+    status = None
+    institution_type = None
+    date_filed = None
+
+    for field in sidefoot:
+        if field['type'] == 'related_metadata':
+            metadata = field['value']['content']
+            for item in metadata:
+                heading = item['value']['heading']
+
+                try:
+                    if heading == 'Court':
+                        court = get_metadata_value(item, heading)
+                    elif heading in ['Institution type', 'Institution']:
+                        institution_type = get_metadata_value(
+                            item,
+                            heading
+                        )
+                    elif heading == 'Status':
+                        listed_status = clean_status(
+                            get_metadata_value(item, heading)
+                        )
+                        if listed_status not in STATUS_OPTIONS:
+                            status = STATUS_OPTIONS[0]
+                        else:
+                            status = listed_status
+                    elif heading.lower() in [
+                        'docket number',
+                        'case number',
+                        'case numbers',
+                        'file number',
+                        'file numbers',
+                        'civil action number'
+                    ]:
+                        docket_number = item['value']['blob']
+                    elif heading == 'Date filed':
+                        date_filed = item['value']['date']
+                    elif (
+                        heading == 'Category' or
+                        heading == 'Topics' or
+                        heading == 'Last updated'
+                    ):
+                        continue
+                except Exception:
+                    return errors.append(
+                        f'Error accessing {heading} metadata for {url}'
+                    )
+        else:
+            sidefoot_fields.append(field)
+
+    if not institution_type:
+        return errors.append(f'Missing Institution type for {url}')
+    if not status:
+        return errors.append(f'Missing Status for {url}')
+    if not docket_number:
+        return errors.append(f'Missing Docket number for {url}')
+
+    statuses = [EnforcementActionStatus(status=status)]
+
+    page.delete()
+
+    eap = EnforcementActionPage(
+        depth=keys['depth'],
+        live=keys['live'],
+        categories=[category],
```

Comment:
```suggestion
        categories=old_categories,
```

I tested this on page id 1431, which has multiple categories, and it seemed to work fine. This way we transfer all applicable categories, and line 70 (`category = CFGOVPageCategory(name=old_categories[0].name)`) is unnecessary.

A quick search seems to show that only 2 pages have multiple categories, and they're both "Civil Action" and "Administrative Proceeding". But better to preserve those than to lose them, since it's simple to do so.

Reply:
Yeah this works for me as well, I'll push up a change with this.